src/components/User/UserFeedbackComponent.jsx

```diff
@@ -76,20 +77,24 @@ export class UserFeedbackComponent extends React.Component {
             />
             <Table.Body>
               {
-                this.props.feedback.map((feedback, index) => (
-                  <TableRowComponent
-                    key={index} //eslint-disable-line
-                    data={feedback}
-                    headings={['reported_by',
+                this.props.feedback.map((feedback, index) => {
+                  feedback.created_at = formatDate(feedback.created_at);
+
+                  return (
+                    <TableRowComponent
+                      key={index} //eslint-disable-line
+                      data={feedback}
+                      headings={['reported_by',
                       'created_at',
                       'report_type',
                       'message']}
-                  >
-                    <Table.Cell>
-                      <ActionComponent />
-                    </Table.Cell>
-                  </TableRowComponent>
-                ))
+                    >
+                      <Table.Cell>
+                        <ActionComponent />
+                      </Table.Cell>
+                    </TableRowComponent>
+                );
```

Comment:
The indentation is off. Add 2 more spaces to align `);` with the `return` keyword